File: cloud-storage-spring-java/src/main/java/com/github/migangqui/cloud/storage/spring/bean/DeleteFileResponse.java
package com.github.migangqui.cloud.storage.spring.bean;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class DeleteFileResponse {
	private boolean result;
	private int status;
	private String cause;
	private Exception exception;
}

File: cloud-storage-spring-kotlin/src/main/kotlin/com/github/sevtech/cloud/storage/spring/model/DeleteFileResponse.kt
package com.github.sevtech.cloud.storage.spring.model

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_EMPTY)
data class DeleteFileResponse(var result: Boolean = false,
                              var status: Int = 0,
                              var cause: String? = null,
                              var exception: Exception? = null
) : BaseResponse(status, cause, exception)